cmd/whoami/cmd.go

```diff
@@ -102,33 +106,37 @@ func run(_ *cobra.Command, _ []string) {
 			os.Exit(1)
 		}
 	}
-	fmt.Printf(""+
-		"AWS Account ID:               %s\n"+
-		"AWS Default Region:           %s\n"+
-		"AWS ARN:                      %s\n"+
-		"OCM API:                      %s\n"+
-		"OCM Account ID:               %s\n"+
-		"OCM Account Name:             %s %s\n"+
-		"OCM Account Username:         %s\n"+
-		"OCM Account Email:            %s\n"+
-		"OCM Organization ID:          %s\n"+
-		"OCM Organization Name:        %s\n",
-		r.Creator.AccountID,
-		awsRegion,
-		r.Creator.ARN,
-		cfg.URL,
-		account.ID(),
-		account.FirstName(), account.LastName(),
-		account.Username(),
-		account.Email(),
-		account.Organization().ID(),
-		account.Organization().Name(),
-	)
+	outputObject := object.Object{
```

Comment:
Not sure if this could just be like this or should have some sort of hierarchy to it. 

e.g. aws.accId, as right now it would only work through ["AWS Account ID"]  notation

Reply:
I m ok with object. Object tbh. This pave the way for other generic implementations as well.